randovania/games/samus_returns/logic_database/Area 3 - Factory Interior.txt

```diff
@@ -697,6 +691,8 @@ Extra - asset_id: collision_camera_030
 > Event - Gamma Awakened; Heals? False
   * Layers: default
   * Event Area 3 (Factory Interior) - Gamma Metroid Upper Awakened
+  > Tunnel to Gamma Arena & Transport to Metroid Caverns East (Bottom)
+      Morph Ball and Movement (Advanced)
```

Comment:
Does this need bombs/damage boosting?

Reply:
This is to go backwards after triggering the Gamma to move. It activates once the crumble is broken which reveals the hidden area, so with careful movement you can hit the block and back out without falling in.